### DISCUSSION TITLE
2 HDMI input?

### DISCUSSION THREAD

**Discussion by ecard0** [tiny-pilot/tinypilot#464] [Q&A] [answered]
Would it be possible to handle 2 HDMI input ?  I would like to control my desktop that has 2 monitor connected.

Thanks, Eric

**Comment by mtlynch** [comment, marked as answer]
Unfortunately, TinyPilot doesn't support this yet.

**Comment by ossiemoore** [comment]
While not ideal, you could actually do this by running two Tiny Pilots.. one for the keyboard/mouse/monitor 1, and a completely separate unit for monitor two. (heh.. was going to say "poor man" dual monitor support, but since you'd have to build/buy two of them, I suppose I should call it the "rich man" approach! :)

**Reply by ecard0** [reply]
I already thought about that approach :)

The problem with 2 monitors, is that application sometime default to the other screen :(
